spec/requests/users_spec.rb

```diff
@@ -51,4 +51,87 @@
       end
     end
   end
+
+  describe '#edit_user_request' do
+    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
+    let(:user) { create :user }
+    let(:modified_user) { attributes_for :user }
+
+    it 'with no logged user - redirects to login' do
+      get edit_user_registration_path
+      expect(response).to redirect_to(new_user_session_path)
+    end
+
+    context 'with logged user' do
+      before do
+        user.avatar.attach(io: File.open(image_path), filename: 'download.jpeg', content_type: 'image/jpeg')
+        sign_in user
+        get edit_user_registration_path, params: { user: modified_user }
```

Comment:
You are not updating the user in this request, so you don't need to send the parameters